stable/nginx-ingress/README.md

```diff
@@ -69,6 +69,7 @@ Parameter | Description | Default
 `controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
 `controller.extraArgs` | Additional controller container arguments | `{}`
 `controller.kind` | install as Deployment, DaemonSet or Both | `Deployment`
+`controller.deploymentAnnotations` | annotations to be added to deployment | `{}`
```

Comment:
Does deploymentAnnotations make sense when it is applied to both the daemonset and the deployment? Not sure if there is something better that we can call this or not.

Reply:
I have implemented on both (deployment and daemonset), Do you think it will be better to separate it to different variables?

Reply:
Eh, I think it is fine.